Relativity.Transfer.Client.md

```diff
@@ -3,6 +3,20 @@
 [![nuget](https://img.shields.io/nuget/v/Relativity.Transfer.Client.svg)](https://www.nuget.org/packages/Relativity.Transfer.Client)
 
 The Relativity Transfer Client .NET SDK package.
+## 7.3.11
+
+### Release Notes
+
+#### Fixed
+
+* Reading cached credentials no longer prolong cached lifetime
```

Comment:
Maybe "To avoid an error of expired credentials, TAPI won't prolong a lifetime of credentials read from the cache. In case of expiration, new credentials will be requested instead"?

Reply:
Done.

Reply:
Approved.